packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -82,6 +84,9 @@ const commands = [
         type: String,
         usage: 'serve [options]',
         description: 'Run the webpack Dev Server',
+        flags: devServer.reduce((cliArgs, { name, type, describe }) => {
```

Comment:
Why we need `reduce` here?

Reply:
`WDS` exports the `cli-flags`, wherein the respective descriptions are available with the key `describe`:-
https://github.com/webpack/webpack-dev-server/blob/2a50946b024b3d0cf7a2609f1491b65da499f5f3/bin/cli-flags.js#L15

https://github.com/webpack/webpack-cli/blob/d0acf3072edd8182c95e37997ac91789da899d66/packages/webpack-cli/lib/groups/runHelp.js#L75